examples/file-upload/src/keys.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-file-upload
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {BindingKey} from '@loopback/core';
+import {RequestHandler} from 'express-serve-static-core';
+
+export const FILE_UPLOAD_SERVICE = BindingKey.create<RequestHandler>(
```

Comment:
![image](https://user-images.githubusercontent.com/950112/76145937-64381c00-60b4-11ea-954c-4a50fa9831aa.png)

Reply:
I see it too. But `npm run build` does not complain.